indra/newview/lluilistener.cpp

```diff
@@ -97,6 +99,41 @@ LLUIListener::LLUIListener():
         "Add menu separator to the [\"parent_menu\"] within the Top menu.",
         &LLUIListener::addMenuSeparator,
         llsd::map("parent_menu", LLSD(), "reply", LLSD()));
+
+    add("setMenuVisible",
+        "Set menu [\"name\"] visibility to [\"visible\"]",
+        &LLUIListener::setMenuVisible,
+        llsd::map("name", LLSD(), "visible", LLSD(), "reply", LLSD()));
+
+    add("defaultToolbars",
+        "todo: defaultToolbars desc",
+        &LLUIListener::restoreDefaultToolbars);
+
+    add("clearToolbars",
+        "Clear all buttons off the toolbars",
+        &LLUIListener::clearAllToolbars);
```

Comment:
Hmm, okay. There's no requirement to name the C++ method the same as its registered string name, and you do use them consistently. It made me pause and scroll back to cross-check when I saw `UI.clearToolbars()`, but this is correct.

Reply:
I'll change it to `clearAllToolbars` for consistency anyway, thanks.